rest-api/tools/import_organizations.py

```diff
@@ -0,0 +1,156 @@
+"""Imports awardees / HPOs, organizations, and sites into the database.
+
+Usage:
```

Comment:
May be worth stating whether or not this script is meant to be idempotent.

Reply:
Done.